test/controllers/EditSetRecordControllerSpec.scala

```diff
@@ -1732,7 +1732,9 @@ class EditSetRecordControllerSpec extends BaseControllerSpec {
   private def givenCreatorIdsArePrepared(expectedEditSetRecord: EditSetRecord)(implicit
     editSetRecordService: EditSetRecordService
   ): ScalaOngoingStubbing[EditSetRecord] =
-    when(editSetRecordService.prepareCreatorIDs(expectedEditSetRecord)).thenReturn(expectedEditSetRecord)
+    when(
+      editSetRecordService.prepareCreatorIDs(any[Seq[Creator]], ArgumentMatchers.eq(expectedEditSetRecord))
```

Comment:
This wouldn't be necessary if we moved it into `EditSetRecordController`.

Reply:
Moving `prepareCreatorIDs` into the controller has broken the tests - mockito fails at runtime in a way I do not understand.

Reply:
Confirmed.